src/pcapi/scripts/beneficiary/remote_import.py

```diff
@@ -151,14 +180,23 @@ def parse_beneficiary_information(application_detail: dict, procedure_id: int) -
             information["phone"] = value
         if label == "Quel est le code postal de votre commune de résidence ?":
             space_free = str(value).strip().replace(" ", "")
-            information["postal_code"] = re.search("^[0-9]{5}", space_free).group(0)
+            try:
+                information["postal_code"] = re.search("^[0-9]{5}", space_free).group(0)
```

Comment:
Je me demande si on ne devrait pas aussi être un peu plus laxiste ici et changer en:
```suggestion
                information["postal_code"] = re.search("[0-9]{5}", space_free).group(0)
```

Reply:
Je ne pense pas : on risque de valider des chaînes qui ne sont pas correctes et casser plus tard dans le code
Meme si le modele pydantic plus loin accepte une chaine optionnelle.

Je préfère qu'on soit stricts ici